### ISSUE TITLE
Description how to enter fastboot is wrong for E5

### ISSUE THREAD

**Issue by Flohack74** [ubports/ubports-installer#345]
* E5 has power and volume buttons on the same side
* Fastboot is not entered with Vol-Down and Power long-presses from any state but by pressing Vol-Up, Down and Power at the same time from the phone being turned OFF

**Comment by Flohack74** [comment]
Plus dont press them too long, it takes max 3 seconds, otherwise it turns off again. Meh.

**Comment by NeoTheThird** [comment]
Urgh, okay, thanks for reporting. Unfortunately the old API can't handle complex instructions like this, so this will stay for now. This will be fixed when we switch to the new API.

**Comment by iozudono** [comment]
With all my respect and my poor English, what Flohacj74 says, at least with the E5LTE, only works to enter in a test mode. To get the fastboot mode I had to maintain pressed the vol-down and the power buttons'. That there are, that it is correct, in the same side.

By other part, when I got this the app recognised the fastboot mode and jumped to asking the oem-unlock, and after giving permissions it crashed again and again... But that was other error I've reported as other issue.
